### DISCUSSION TITLE
Welcome to docker-squid Discussions!

### DISCUSSION THREAD

**Discussion by b4tman** [b4tman/docker-squid#39] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hpcmtint** [comment]
hello, I've downloaded and got your container running, however, I need this added to the config 

```
http_port 3128 proxy-protocol
proxy_protocol_access allow localhost
proxy_protocol_access deny all
```
get this error 

```
+ '[' -e /var/run/squid/squid.pid ]
+ /usr/sbin/squid -f /etc/squid/my-squid.conf --foreground -z
2025/04/08 12:59:35| Processing Configuration File: /etc/squid/my-squid.conf (depth 0)
2025/04/08 12:59:35| FATAL: Unknown http_port option 'proxy-protocol'.
2025/04/08 12:59:35| Not currently OK to rewrite swap log.
2025/04/08 12:59:35| storeDirWriteCleanLogs: Operation aborted.
**2025/04/08 12:59:35| FATAL: Bungled /etc/squid/my-squid.conf line 1: http_port 3128 accel proxy-protocol**
2025/04/08 12:59:35| Squid Cache (Version 6.13): Terminated abnormally.
CPU Usage: 0.008 seconds = 0.008 user + 0.000 sys
Maximum Resident Size: 40512 KB
Page faults with physical i/o: 0
```

please advise and I need this to work.
